### DISCUSSION TITLE
LDAP--In Progress

### DISCUSSION THREAD

**Discussion by abdullah4142** [nilsteampassnet/TeamPass#3666] [General]
Hello, I installed version 3.0.3, but when I try to log in from Active Directory with LDAP, it stays as "In Progress" and does not log in. What should I do ? Could you help ? Thanks in advance.

**Comment by chises** [comment]
Hello,
I has the same issue. did you import the user? After that it worked for me:
![image](https://github.com/nilsteampassnet/TeamPass/assets/10805806/2098c22c-ccc1-47be-9a96-c2f38e9d1497)

**Comment by abdullah4142** [comment]
Hello, unfortunately, the transfer cannot be made, it says "loading", but
the process is not completed.



*Abdullah DÖLEK*


chises ***@***.***>, 16 Haz 2023 Cum, 23:16 tarihinde şunu
yazdı:

**Comment by ismaelvillalba** [comment]
> Hola, instalé la versión 3.0.3, pero cuando trato de iniciar sesión desde Active Directory con LDAP, queda como "En progreso" y no inicia sesión. ¿Qué debo hacer? Podrías ayudar ? Gracias de antemano.

Hi Abdullah! I'm having the same problem... could you find the solution? How did you configure the service for LDAP? Thank you so much!
